src/page/PostList/components/LineLike/LineLike.tsx

```diff
@@ -16,7 +16,9 @@ type LineLikeProps = {
 }
 export const LineLike = ({postId}: LineLikeProps) => {
     
-    const { posts, onLikePost, onDislikePost, onFavoritePost } = usePosts()
+    const { getInfoOnePost, onLikePost, onDislikePost, onFavoritePost } = usePosts()
+
+    const infoPost = getInfoOnePost(postId-1);
```

Comment:
здесь то же самое про id
```suggestion
    const infoPost = getInfoOnePost(postId);
```